src/helpers/indexing-helper.php

```diff
@@ -79,7 +79,7 @@ public function set_first_time( $is_first_time_indexing ) {
 	 *
 	 * @return void
 	 */
-	public function get_first_time() {
+	public function is_first_time() {
```

Comment:
much better - but first time still doesn't mean anything (naming is HARD)
I've heard the term initial indexing fly past last week - does that apply here? is_initial_indexing?

Reply:
I think that is a good suggestion, will update.